frontend/src/editor/automerge_websocket_provider.ts

```diff
@@ -1,53 +1,67 @@
-import { Observable } from 'lib0/observable';
-import { Change } from '@automerge/automerge';
+import { useEffect, useMemo, useRef } from 'react';
```

Comment:
This file should also be renamed, since it no longer contains the `AutomergeWebsockerProvider`